recipe/meta.yaml

```diff
@@ -1,229 +1,317 @@
-{% set version = "3.0.2" %}
+{% set version = "3.6.0" %}
 
 package:
   name: gdal-split
   version: {{ version }}
 
 source:
   url: http://download.osgeo.org/gdal/{{ version }}/gdal-{{ version }}.tar.xz
-  sha256: c3765371ce391715c8f28bd6defbc70b57aa43341f6e94605f04fe3c92468983
+  sha256: f7afa4aa8d32d0799e011a9f573c6a67e9471f78e70d3d0d0b45b45c8c0c1a94
   patches:
-    # BUILT_AS_DYNAMIC_LIB.
-    - patches/windowshdf5.patch  # [win]
-    # Use multiple cores on Windows.
-    - patches/multiprocessor.patch  # [win]
-    # disable 12 bit jpeg on Windows as we aren't using internal jpeg
-    - patches/disable_jpeg12.patch  # [win]
-    # Fix missing include
-    - patches/include-limits-missing.patch  # [linux]
+    - 0001-Improve-numpy-fixing.patch
+    - 0002-Parquet-make-sure-that-ArrowLayer-destructor-is-avai.patch
 
 build:
-  number: 6
-  # Has never been built for s390x
-  skip: True  # [s390x]
+  number: 14
+  # never be built on s390x
+  skip: True  # [linux and s390x]
+  skip_compile_pyc:
+    - "share/bash-completion/completions/*.py"
   ignore_run_exports:
     - krb5  # [win]
 
 requirements:
   build:
-    - cmake  # [win]
-    - pkg-config >=0.21  # [not win]
+    - cmake
+    - pkg-config >=0.21      # [not win]
+    - make                   # [not win]
+    - ninja                  # [win]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - make  # [not win]
-    - libtool  # [not win]
-    - m2-patch  # [win]
-    - patch  # [linux]
+    - sysroot_linux-64 2.17  # [linux64]
+    - patch                  # [not win]
+    - m2-patch               # [win]
+
   host:
+    - blosc
     - cfitsio
-    - curl
     - expat
     - freexl
     - geos
     - geotiff
-    - giflib  # [not win]
+    - giflib   # [not win]
     - hdf4
     - hdf5
     - icu
     - jpeg
-    - json-c  # [not win]
+    - json-c   # [not win]
     - kealib
+    - lerc
     - libcurl
-    - libdap4  # [not win]
+    - libdeflate
+    - libiconv
     - libkml
     - libnetcdf
     - libpng
     - libpq
     - libspatialite
     - libtiff
     - libuuid  # [linux]
-    # exclusions due to https://bugs.launchpad.net/lxml/+bug/1928795
-    - libxml2 !=2.9.11, !=2.9.12
-    - m2w64-xz  # [win]
+    - libwebp-base
+    - libxml2
+    - lz4-c
     - openjpeg
     - openssl
-    - pcre
-    - poppler <=0.81  # [not win]
+    - pcre2
+    - poppler
     - postgresql
     - proj
+    - qhull
```

Comment:
There were noticed issues with qhull https://github.com/conda-forge/qgis-feedstock/issues/284#issuecomment-1356490896

Reply:
yes, but I couldn't reproduce ... not sure here, but not sure how much we need to be concerned.  we do not provide qgis AFAIK, and common testsuite hasn't shown any regressions so far